confgenerator/logging_processors.go

```diff
@@ -364,3 +368,66 @@ end`, lua))...)
 func init() {
 	LoggingProcessorTypes.RegisterType(func() LoggingProcessor { return &LoggingProcessorExcludeLogs{} })
 }
+
+// A LoggingProcessorGceMetadataAttributes annotates logs with labels
+// corresponding to instance attributes from the GCE metadata server.
+type LoggingProcessorGceMetadataAttributes struct {
+	ConfigComponent `yaml:",inline"`
+	Include         []string `yaml:"include" validate:"dive,gt=0,glob"`
```

Comment:
These validations will never be used, AFAICT

Reply:
There's no way to enable that without also enabling use in user configs.

Reply:
Removed the validations.